### ISSUE TITLE
Fix bundle upload example

### ISSUE THREAD

**Issue by sfontana** [googleads/googleads-python-lib#243]
https://github.com/googleads/googleads-python-lib/blob/master/examples/adwords/v201710/misc/upload_media_bundle.py tries to upload a bundle which not valid because it "has modified clickability" which is not supported.
The example needs to be updated with a valid bundle.

Also: the error returned is rather confusing since it says 
`suds.WebFault: Server raised fault: '[OperationAccessDenied.ACTION_NOT_PERMITTED @ class com.google.ads.api.serviceimpls.campaignmgmt.media.UploadAction]'`

**Comment by msaniscalchi** [comment]
Hello Stefano,

Thanks for the report! I'll look into what changes we may need to make to that sample.

Regarding the error, could you provide some additional feedback on what is confusing about it? At first glance, it seems reasonable that an OperationAccessDenied.ACTION_NOT_PERMITTED error was raised by the API, given that the action the sample is attempting to do is invalid. Were you expecting to see something different?

Regards,
Mark

**Comment by sfontana** [comment]
Sure,

IMHO the error should say that the value I'm using for the action is not permitted, not the action itself.
When I first saw the error I thought there was something wrong with my refresh token.

BTW when using a bulk job for the same operation we do get a different error 
``
OrderedDict([('@xmlns:xsi', 'http://www.w3.org/2001/XMLSchema-instance'), ('@xsi:type', 'MediaBundleError'), ('fieldPath', 'operations[1].operand.ad.templateElements[0].fields[0].fieldMedia.data'), ('fieldPathElements', [OrderedDict([('field', 'operations'), ('index', '1')]), OrderedDict([('field', 'operand')]), OrderedDict([('field', 'ad')]), OrderedDict([('field', 'templateElements'), ('index', '0')]), OrderedDict([('field', 'fields'), ('index', '0')]), OrderedDict([('field', 'fieldMedia')]), OrderedDict([('field', 'data')])]), ('trigger', None), ('errorString', 'MediaBundleError.INVALID_MEDIA_BUNDLE'), ('ApiError.Type', 'MediaBundleError'), ('reason', 'INVALID_MEDIA_BUNDLE')])
`` 
which is clearer to me at least.

Cheers

**Comment by msaniscalchi** [comment]
As an FYI, I took another look at this recently. I wasn't able to reproduce the original error message you sent--I consistently receive a MediaBundleError.INVALID_MEDIA_BUNDLE.

That said, I've been able to reproduce this in other client libraries. It seems like something may have changed and broken the sample after it was originally published--e.g. perhaps the content at the URL. I'll continue to investigate and expect this to be resolved by the next release.

-- Mark
